quizbizz/src/pages/Homepage/HeroSection/HeroUserInfo/UserProgressContainer/ProgressItem.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const ProgressItem = ({
+  progressItemTitle,
+  progressItemSubTitle,
+  progressItemIcon,
+}) => {
+  return (
+    <div className="progress-item">
+      <span>{progressItemIcon()}</span>
```

Comment:
suggestion: remove () in progressItemIcon() if not in use

Reply:
That is used because that is a icon